### PR TITLE
add more test and enable injectedStoredValue for check/uncheck action

### PR DESCRIPTION
**Pull request by johnnie502**
This will add injectedStoredValue function to check/uncheck action, and add more tests for the checkbox

### CODE REVIEW

**Comment by johnnie502** [comment]
!force code review

**Comment by medology-bugman** [comment]
Paging @Chekote to review the back-end work in this pull request!
Paging @awesomeousPrime to review the testing work in this pull request!

**Comment by awesomeousPrime** [comment]
@johnnie502 testing review 👍 88a08db

**Comment by johnnie502** [comment]
!reassign back-end @jose-ochoa

**Comment by medology-bugman** [comment]
Paging @jose-ochoa to review the back-end work in this pull request!

File: src/Behat/FlexibleMink/Context/FlexibleContext.php [hunk 1]
```diff
@@ -153,6 +154,19 @@ public function checkOption($locator)
 
     /**
      * {@inheritdoc}
+     */
+    public function uncheckOption($locator)
```
Comment (jose-ochoa) [inline src/Behat/FlexibleMink/Context/FlexibleContext.php]: Where is the doc block inheriting from if this is new?

- [ ] Done
- [ ] Approved

**Pull request merged**